internal/server/spanner/query.go

```diff
@@ -42,12 +42,33 @@ var (
 
 // SQL / GQL statements executed by the SpannerClient
 var statements = struct {
+	getPropsBySubjectID             string
+	getPropsByObjectID              string
 	getEdgesBySubjectID             string
 	getEdgesByObjectID              string
 	getObsByVariableAndEntity       string
 	getObsByVariableEntityAndDate   string
 	getLatestObsByVariableAndEntity string
 }{
+	getPropsBySubjectID: `
+	SELECT
+		DISTINCT(predicate),
```

Comment:
I ordered them this way since it seems like distinct needs to be applied to the first selected columns (there was a parsing error when I reversed the order)

Reply:
Nevermind, sorry! It looks like the order doesn't matter, so I was able to flip the columns :)